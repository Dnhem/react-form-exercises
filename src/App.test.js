import { render, screen } from "@testing-library/react";
import App from "./App";

// test("it renders without crashing", () => {
//   render(<TodoList />);
// });

// test("it matches snapshot", () => {
//   const { asFragment } = render(<TodoList />);
//   expect(asFragment()).toMatchSnapshot();
// });

// test("playing with queries", () => {
//   const { getByText } = render(<TodoList />);
//   console.log(getByText("Todo List âœ…"));
// });
