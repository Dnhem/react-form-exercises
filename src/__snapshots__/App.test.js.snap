// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Todo List âœ…
    </h1>
    <form>
      <label
        for="todo"
      >
        Enter new todo:
      </label>
      <input
        id="todo"
        name="todo"
        type="text"
        value=""
      />
      <button>
        Add
      </button>
    </form>
    <ul />
  </div>
</DocumentFragment>
`;
